service/k8sclient/k8sclient.go

```diff
@@ -2068,19 +2084,20 @@ func (c *K8sClient) getPSMDBSpec112Plus(params *PSMDBParams, extra extraCRParams
 					MultiAZ: psmdb.MultiAZ{
 						Affinity: extra.affinity,
 					},
-					Expose: extra.expose,
 				},
 				Mongos: &psmdb.MongosSpec{
 					Size: params.Size,
 					MultiAZ: psmdb.MultiAZ{
 						Affinity: extra.affinity,
 					},
 					Expose: psmdb.MongosExpose{
-						ExposeType: common.ServiceTypeLoadBalancer,
+						ExposeType: extra.expose.ExposeType,
```

Comment:
@BupycHuk Here's the expose logic for PSMDB sharded clusters

Reply:
Yeah, but I meant `extra.expose.Enabled`, not `extra.expose.ExposeType`

Reply:
[v1.9.0](https://github.com/percona/percona-server-mongodb-operator/releases/tag/v1.9.0)
It was enabled by default in PSMDB 1.9. I think that we can drop it, right?

Reply:
Could you point to the task, please?
As far as I remember users want to be able to not expose their clusters outside of the kubernetes cluster.
If we ignore expose parameters it's a bug.